packages/mobile/src/screens/user-list-screen/Tip.tsx

```diff
@@ -0,0 +1,37 @@
+import { StringWei } from 'audius-client/src/common/models/Wallet'
+import { View } from 'react-native'
+
+import IconTip from 'app/assets/images/iconTip.svg'
+import { Audio, Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+const useStyles = makeStyles(() => ({
+  root: {
+    flexDirection: 'row',
+    alignItems: 'center'
+  }
+}))
+
+type TipProps = {
+  amount: StringWei
+}
+
+export const Tip = (props: TipProps) => {
+  const { amount } = props
+  const styles = useStyles()
+  const { neutralLight4 } = useThemeColors()
+
+  return (
+    <View style={styles.root}>
+      <IconTip fill={neutralLight4} height={14} width={14} />
+      <Text color='neutralLight4'>
```

Comment:
Does it make sense to wrap like this or should we just have `color='neutralLight4'` twice

Reply:
i was trying something new out, but agreed it seems quite redundant here!